examples/basic/test/basic.test.ts

```diff
@@ -8,7 +8,7 @@ test('Math.sqrt()', () => {
   expect(Math.sqrt(2)).toBe(Math.SQRT2)
 })
 
-test('JSON', () => {
+test.skip('JSON', () => {
```

Comment:
Please, don't modify tests in basic examples. They are used in https://vitest.new